test/libsolidity/syntaxTests/inlineAssembly/use_msize_without_optimizer.sol

```diff
@@ -8,3 +8,4 @@ contract C {
 // ====
 // optimize-yul: false
 // ----
+// SyntaxError 6553: (52-101): The msize instruction cannot be used when the Yul optimizer is activated because it can change its semantics. Either disable the Yul optimizer or do not use the instruction.
```

Comment:
Note that this PR effectively disallows `msize` because users can no longer disable the Yul optimizer.

What should I do about it? I suspect that with the minimal optimization level `msize` is actually still usable, so I could make the check account for that. I'm not sure if we even care about it. The alternative is to just adjust the text to remove the second sentence.

Reply:
Just don't change anything for Yul sources that contain ``msize``. I.e. *don't* try to run any optimizer steps or stack-to-memory or the optimized code transform on such sources - just have them compile exactly as they do now.

Reply:
Done.